### ISSUE TITLE
Hangs on "Prerequisite checks"

### ISSUE THREAD

**Issue by treshugart** [pikapkg/pack#37]
Hey everyone. Rad tool you got here.

I'm using Windows (maybe this is the problem? usually is) and when I run `pack publish`, it hangs after asking if I want to publish it as a public scoped package (I use the default "Y") on "[1/7] ✨  Prerequisite checks...". There's no NPM errors or anything being reported, it just seems to run indefinitely.

I've run this in several shells: cmd, hyper, vscode terminal (via powershell), powershell. I have not tried using admin privs (hopefully this isn't necessary, and would probably just error anyways if it req'd it).

I don't really have much more info that I could give up front. Sorry about that.

**Comment by FredKSchott** [comment]
I believe that this is the same issue as https://github.com/pikapkg/pack/issues/2 This issue seems tricky since it seems to be an issue with npm itself on Windows ¯\_(ツ)_/¯

Will look more into a fix when I can

**Comment by FredKSchott** [comment]
@treshugart thanks for filing! Fixed by #38, going out in a patch in a little bit

**Comment by treshugart** [comment]
Oh rad, I was going to have a deeper dive this morning to see if I could do anything. Thanks!

**Comment by Saeris** [comment]
Still experiencing a hang on prereq checks on Windows 10, also tried a number of different shells. I tried running the git commands manually to see if there were any errors with those specifically, but all of those completed without error.

I don't know what could be going wrong, because the output isn't very verbose, so I don't even know which step it's hanging on specifically.

Given that I only have windows machines to work with, this issue is preventing me from being able to use this for anything other than running builds. Only option I have is to boot up a linux VM and try publishing from that.

**Comment by Saeris** [comment]
Okay, after much trial and error I was able to successfully publish my package by using a CodeSandbox container. Along the way I found some bumps in the road I wasn't experiencing on Windows:

`npm access ls-collaborators` failed because I had not yet ever published the package to NPM, so it received an error back telling me the package was not found.

The remaining hiccups were all git related, since I had to manually setup my repository within the sandbox container, set my global git config settings, and make sure my branch was clean.

After successfully publishing once, I tried again on Windows and it still hung on prereqs. So no closer to figuring out the problem but at least I've published my package for now.
